3.3/test/django-test-apps/no-static/run.py

```diff
@@ -0,0 +1,2 @@
+import os
+os.execl('./manage.py', './manage.py', 'runserver', '0.0.0.0:8080')
```

Comment:
I'd rather avoid showing this as an option for users to run django app. Generally speaking using dev server is discouraged, I'd prefer this app be run in gunicorn or differently as a standalone.